src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -94,6 +94,7 @@ __vsc_preexec() {
 
 __vsc_update_prompt
 
+__vsc_original_ifs="$IFS"
```

Comment:
I'm really not an expert, but shouldn't we be saving this on each call to `__vsc_prompt_cmd_original` rather than at the top level?

Reply:
I wanted to avoid that since this will run every time enter is pressed. since all of the startup files get run at the top of this file, I believe this will capture any changes that are made to `IFS` in those. 

What case are you thinking of that this will not cover?

Reply:
I'm sure you understand this code's context better than I do. It just seems like if I, at the shell, write `IFS=bla` and hit enter, it will get overwritten by the next prompt back to whatever my startup files had set?

Reply:
I see - yes I was under the assumption that `IFS` will only be modified via people's rc files. You have a good point that a user may set it directly and that wouldn't get picked up.

Reply:
I think you are right here. Thanks for the thoughts - I am pretty new to bash scripting